datasets/modeled/buildings/dataset.json5

```
@@ -1,78 +1,80 @@
+  origin_creator: "Margaret Pigman",
   origin_organization: "LBL",
   origin_contributors: [
-    "Ty Frazier",
+    "Jared Langevin",
     "Jeff Deason",
     "Chioke Harris",
     "Henry Horsey",
     "Andrew Parker",
-    "Margaret Pigman",
     "Handi Chandra Putra",
   ],
-  origin_project: "Scout",
-  origin_date: "2023-12-18",
-  origin_version: "v0.9.1",
-  data_source: "scout",
-  source: "Scout (v0.9.1 — https://github.com/trynthink/scout/releases/tag/v0.9.1) simulation run on 12/20/2023, with data from ResStock (2021/resstock_amy2018_release_1) and ComStock (2021/comstock_amy2018_release_1)",
+  origin_project: "Buildings Standard Scenarios",
+  origin_date: "2025-01-31",
+  origin_version: "v1",
+  data_source: "bss",
+  source: "B-SS public data release, with data from ResStock (2024/resstock_amy2018_release_2) and ComStock (2025/comstock_amy2018_release_2)",
   data_classification: "low",
   tags: [
-    "residential",
-    "commercial",
+    "res",
+    "com",
+    "buildings",
+    "Scout",
+    "ResStock",
+    "ComStock"
   ],
   trivial_dimensions: [
-    "scenario",
     "subsector",
     "weather_year",
   ],
   data_schema: {
     table_format: {
-      format_type: "unpivoted",
-      value_column: "value",
+      format_type: "pivoted",
+      pivoted_dimension_type: "metric",
     },
     data_schema_type: "one_table",
   },
 
   dimensions: [
     {
       type: "geography",
-      name: "emm_region",
-      file: "dimensions/regions.csv",
+      name: "county",
+      file: "dimensions/county.csv",
       module: "dsgrid.dimension.standard",
       class: "GeographyDimensionBaseModel",
-      description: "25 US EIA Electricity Market Module (EMM) regions with county-level representation based on equal weighting for each county within an EMM region (https://www.eia.gov/outlooks/aeo/pdf/nerc_map.pdf)",
+      description: "US counties for lower 48 states",
     },
     {
       type: "metric",
       name: "building_end_use",
       file: "dimensions/enduses.csv",
       module: "dsgrid.dimension.standard",
       class: "EnergyEndUse",
-      description: "Scout residential and commercial end uses (N=13) based on EIA Annual Energy Outlook",
+      description: "Scout residential and commercial end uses (N=9) based on EIA Annual Energy Outlook",
     },
     {
       type: "model_year",
       name: "building_model_year",
       file: "dimensions/model_years.csv",
       module: "dsgrid.dimension.standard",
       class: "ModelYear",
-      description: "Current year 2023 and future snapshot years 2030 and 2050",
+      description: "2024 and every 5 years from 2025 through 2050",
     },
     {
       type: "scenario",
       class: "Scenario",
-      name: "high_scenario",
-      description: "IEF 2023 High-Resolution Demand Scenarios; Limited to a single, high scenario",
-      file: "../../../project/dimensions/scenarios.csv",
+      name: "building_scenarios",
```

```suggestion
      name: "building_scenario",
```

Suggesting making this singular instead of plural, similar to "county," "building_model_year," etc.

---

there are multiple scenarios and model years

---

Yes, but these names are essentially query names, so my thought process is, "What looks best in a SQL statement?" E.g., `SELECT building_scenario FROM ...` or `SELECT building_scenarios FROM ...`.